package.json

```diff
@@ -515,9 +515,15 @@
         "go.buildTags": {
           "type": "string",
           "default": "",
-          "description": "The Go build tags to use for all commands that support a `-tags '...'` argument",
+          "description": "The Go build tags to use for all commands, excluding when running tests (see testTags), that support a `-tags '...'` argument",
```

Comment:
The description now is a little confusing. Can we move the exclude statement to the end? How about the below?

`The Go build tags to use for all commands that support a `-tags '...'` argument. When running tests, `go.testTags` will be used instead if it was set.`

Reply:
That does sound better :)